packages/block-library/src/navigation-link/index.php

```diff
@@ -171,7 +194,7 @@ function render_block_core_navigation_link( $attributes, $content, $block ) {
 
 	$html .= '</span>';
 
-	$html .= '</a>';
+	$html .= '</' . $item_tag . '>';
 	// End anchor tag content.
 
 	if ( $block->context['showSubmenuIcon'] && $has_submenu ) {
```

Comment:
Not specific to this PR but sometimes it generates a notice.

```suggestion
	if ( isset( $block->context['showSubmenuIcon'] ) && $block->context['showSubmenuIcon'] && $has_submenu ) {
```